clusters/app.ci/trt/admin_monitor_prowjob_rbac.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Service Accounts for TRT Aggregator Use
+    openshift.io/display-name: TRT CI Aggregator
+  name: trt-aggregator
```

Comment:
I wouldn't have thought an SA in one namespace could bind to a role in another, but if it works then that's good.